sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1648,14 +1639,19 @@ private void updateCaseWithBulkData(
 	@RightsAllowed({
 		UserRight._CASE_CREATE,
 		UserRight._CASE_EDIT })
-	public CaseDataDto save(@Valid CaseDataDto dto, boolean handleChanges, boolean checkChangeDate, boolean internal, boolean systemSave)
+	public CaseDataDto save(@Valid CaseDataDto dto, boolean handleChanges, boolean checkChangeDate, boolean internal)
 		throws ValidationRuntimeException {
 
 		Case existingCase = service.getByUuid(dto.getUuid(), true);
 		FacadeHelper.checkCreateAndEditRights(existingCase, userService, UserRight.CASE_CREATE, UserRight.CASE_EDIT);
 
-		if (!systemSave && internal && existingCase != null && !service.isEditAllowed(existingCase)) {
-			throw new AccessDeniedException(I18nProperties.getString(Strings.errorCaseNotEditable));
+		if (existingCase != null && internal) {
+			EditPermissionType editPermission = service.getEditPermissionType(existingCase);
+			if (editPermission == EditPermissionType.OUTSIDE_JURISDICTION) {
```

Comment:
I've found a corner case when we have multiple problems: Outside jurisdiction and not enough user rights.
At first I've tried the merge without all the necessary user rights ( the message was correct and in the logs we can see the missing user righst):
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/fd221c39-9af5-42c5-bd4b-99cf265747f8)

After that I've changed the EditPermission Type to be OUTSIDE_JURISDICTION:
![OutsideJurisdictionAndNotEditable](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/67006f05-ec41-47c6-a87a-e8955b7fc92a)

The message that the entity is outside the jurisdiction appears but the other problem ( missing user rights) will not be displayed as it is requested in one of the acceptance criterias:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/96ac4d72-f762-4dc2-8ea3-9baa2da186a8)

Reply:
@leventegal-she we've discussed in the daily the missing acceptance criteria and @MateStrysewske agreed that is not that necessary to display multiple problems so there is no need for any changes.